E2E/pageObjects/game.page.ts

```diff
@@ -133,6 +143,26 @@ export = {
     game.voteIsVisible(chessPiece, tile, value);
   },
 
+  async openPopUp () {
```

Comment:
This method should take parameter that determine, which tile should get it's popup opened. I would like to see something like this:
```
openPopup({ row: 1, col: 6 });
```